src/core/device.c

```diff
@@ -285,6 +285,18 @@ static int device_add_udev_wants(Unit *u, struct udev_device *dev) {
         }
 }
 
+static bool device_is_bound_by_mounts(struct udev_device *dev) {
+        const char *bound_by;
+
+        assert(dev);
+
+        bound_by = udev_device_get_property_value(dev, "SYSTEMD_BIND_MOUNTS");
+        if (!bound_by)
+                return false;
+
+        return parse_boolean(bound_by) != 0;
```

Comment:
probably should be > 0, not != 0. On parse error we should consider it off I think...